packages/components/src/components/dropdown/items.jsx

```diff
@@ -1,8 +1,8 @@
 import classNames          from 'classnames';
 import PropTypes           from 'prop-types';
 import React               from 'react';
-import { Popover }         from 'App/Components/Elements/Popover';
-import { getCurrencyName } from '_common/base/currency_base';
+import Popover             from 'Components/popover';
+import './list.scss';
```

Comment:
@reza-fs can you please move the css import into index.js file where we import item.jsx